src/components/Comments/hooks/useCommentAction.js

```diff
@@ -32,6 +33,9 @@ const useCommentAction = () => {
         if (params.getComment && typeof params.getComment === 'function') {
           params.getComment();
         }
+        if (params.getCommentParent && typeof params.getCommentParent === 'function') {
+          params.getCommentParent();
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a function call `params.getCommentParent()`. Ensure that this function is always available and correctly implemented where this hook is used, to avoid potential errors or unexpected behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->